src/documentation/sdks/python-custom-io.md

```diff
@@ -1,11 +1,11 @@
 ---
 layout: default
-title: "Beam Custom Sources and Sinks for Python"
+title: "Custom Sources and Sinks for Python"
```

Comment:
> I'm not sure about "Python SDK", would we need to say "Custom Sources and Sinks in Apache Beam SDK for Python" which seems long? Thoughts? Or could axe Python from the title entirely. Or "Python Custom Sources and Sinks"?

Ack on this concern; we shouldn't make it too long.

I'd argue for:
* The page title (the thing that goes into `title:`) should (almost) always be `Apache Beam XXX`. Very rarely `XXX` is fine, but it shouldn't be `Beam XXX`.
* The H1 heading (the thing that goes into `#`) can be `XXX` or `Apache Beam XXX`, never `Beam XXX`. Repeating Beam is not necessary (given that context is already established with the logo, menus, etc.).
* Avoid mixing "Python" and "Python SDK". These aren't sources for Python; they are sources in the Beam's Python SDK -- they don't apply generally to Python. We colloquially use "Python" all the time, but in writing should almost always be "Python SDK".